### ISSUE TITLE
Not copying

### ISSUE THREAD

**Issue by Muadiv** [liulex/CopyAsHtml#3]
When I try to copy, I can see this error: 

```
Traceback (most recent call last):
  File "C:\Program Files\Sublime Text 3\sublime_plugin.py", line 1052, in run_
    return self.run()
  File "C:\Users\XXXX\AppData\Roaming\Sublime Text 3\Installed Packages\Copy as HTML.sublime-package\CopyAsHtml.py", line 84, in run
  File "C:\Users\XXXX\AppData\Roaming\Sublime Text 3\Installed Packages\Copy as HTML.sublime-package\CopyAsHtml.py", line 341, in run
  File "C:\Users\XXXX\AppData\Roaming\Sublime Text 3\Installed Packages\Copy as HTML.sublime-package\CopyAsHtml.py", line 155, in setup
  File "C:\Users\XXXX\AppData\Roaming\Sublime Text 3\Installed Packages\Copy as HTML.sublime-package\lib/color_scheme_matcher.py", line 72, in __init__
  File "./python3.3/plistlib.py", line 104, in readPlistFromBytes
  File "./python3.3/plistlib.py", line 76, in readPlist
  File "./python3.3/plistlib.py", line 378, in parse
xml.parsers.expat.ExpatError: not well-formed (invalid token): line 1, column 0

```

And is not copying the text at all, not even with format, is not copying. 
I'm using ST3 on Windows10

Thanks in advance.

**Comment by liulex** [comment]
Thanks for reporing. I will fix it soon. See #2 .

**Comment by liulex** [comment]
Fixed with v2.0.0.
